### PR TITLE
Create PULL_REQUEST_TEMPLATE.md

### PR DESCRIPTION
**Pull request by adeolaadedeji**
Created a pull request file/Template

### CODE REVIEW

**Comment by on2onyekachi** [comment]
Awesome! @adeolaadedeji 

You'll need to site the issue number associated with this PR.

**Comment by adeolaadedeji** [comment]
Thank you @on2onyekachi for the feedback, i have  stated the issue assigned to me.

**Comment by adeolaadedeji** [comment]
Thank you @on2onyekachi for the feedback, i have  stated the issue assigned to me.

**Comment by on2onyekachi** [comment]
oh Sorry, @adeolaadedeji.  Not in the General chat.  you need to state it in the description of this PR.

**Comment by adeolaadedeji** [comment]
Okay @on2onyekachi , thanks. This was the fto assigned to me.......[publiclab/infragram#363](https://github.com/publiclab/infragram/issues/363)

**Comment by on2onyekachi** [comment]
Okay @adeolaadedeji 

> Okay @on2onyekachi , thanks. This was the fto assigned to me.......[publiclab/infragram#363](https://github.com/publiclab/infragram/issues/363)

 Great!, now you need to make an edit and add just the issue number only to the description of your PR. It should be in the format `fixes #0000.` Where #0000 is the issue number.

**Comment by adeolaadedeji** [comment]
Hi @on2onyekachi , this has been worked on. kindly review

**Comment by on2onyekachi** [comment]
@adeolaadedeji I think you made a change to the file itself. Please revert those changes. I dropped you a private message on how to edit the description itself.

**Comment by adeolaadedeji** [comment]
Thank you for the guidelines @on2onyekachi, i have worked on it. Can you please check it now?

**Comment by on2onyekachi** [comment]
@adeolaadedeji Yeah, this is better! now. it shows a commit message that references the raised issue pointing to this PR.

**Comment by adeolaadedeji** [comment]
@on2onyekachi , thank you lots..

**Review by TildaDares — CHANGES_REQUESTED** [review]
@adeolaadedeji You need to copy the markdown and not just the text. Can you copy the contents of this file https://github.com/publiclab/plots2/edit/main/.github/PULL_REQUEST_TEMPLATE.md and paste again? Thanks!!

**Comment by adeolaadedeji** [comment]
Thank you @TildaDares , i have worked it. Kindly review

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+<!-- Add a short description about your changes here-->
+
+Fixes #0000 <!--(<=== Add issue number here)-->
+
+Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!
+
+* [ ] PR is descriptively titled 📑 and links the original issue above 🔗
+* [ ] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
```
Comment (TildaDares) [inline .github/PULL_REQUEST_TEMPLATE.md:8]:
```suggestion
* [ ] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR

**Review by TildaDares — CHANGES_REQUESTED** [review]
Hi @adeolaadedeji, can you make that small correction? Thanks!!

**Comment by adeolaadedeji** [comment]
Hi @TildaDares , changes effected, kindly review. Thank you.

**Review by TildaDares — APPROVED** [review]
Thanks @adeolaadedeji. Great work!!

**Comment by TildaDares** [comment]
Congratulations @adeolaadedeji 🎉

**Comment by adeolaadedeji** [comment]
Thank you so much @TildaDares .

**Pull request merged**